src/meta/src/hummock/compaction_scheduler.rs

```diff
@@ -214,6 +199,36 @@ where
                     continue 'send_task;
                 }
             }
+
+            // 2.3 Send the compaction task.
+            if let Err(e) = compactor
+                .send_task(Task::CompactTask(compact_task.clone()))
+                .await
+            {
+                tracing::warn!(
+                    "Failed to send task {} to {}. {:#?}",
+                    compact_task.task_id,
+                    compactor.context_id(),
+                    e
+                );
+                // Cancel the task at best effort
+                compact_task.task_status = false;
+                if let Err(e) = self
+                    .hummock_manager
+                    .report_compact_task(compactor.context_id(), &compact_task)
+                    .await
+                {
+                    tracing::warn!("Failed to cancel task {}. {:#?}", compact_task.task_id, e);
```

Comment:
I think it shall be `error!` log.
The only fail reason is caused by IO (meta-storage).

Reply:
fixed